docs/source/user_guide/troubleshooting.rst

```diff
@@ -54,6 +65,7 @@ Or in your :file:`~/.condarc` file:
   ...
   channels:
     - ...
+    - defaults  # BAD! Remove this if it exists.
```

Comment:
What about this? It does deactivate the `defaults` altogether so maybe not the best.
```suggestion
default_channels: []
channels:
  - conda-forge
```